android/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,13 +1,15 @@
-• GPX import support
-• New OpenStreetMap data as of June 2
-• MRGS and UTM coordinates support (tap on coordinates to change)
-• Improved car and bicycle routing
-• Fixed bug that prevented upload of OSM edits when on battery power
-• Now compass works on devices without gyroscope
-• Fixed missing Wiki descriptions
-• Decreased TTS (Text-To-Speech) voice speed
-• Added Fish Pass
-• Fixed funicular and parcel locker icons
-• Corrected be, eu, en, en-GB, and uk translations
+• Fresh OpenStreetMap data as of July 7, 2023
+• Fix several crashes
```

Comment:
What crashes? How does this information help users?

Reply:
Please complain to Google Play that limited Whats New to 500 characters. I would put all the detailed info about every crash into the release notes, but unfortunately then we don't have space left for other important improvements.

In this way, if some users will still have crashes, they may contact us.